docs/OCP_CI_Tutorials/Scenario_Development/Scenario_Development_Guide.md

```diff
@@ -238,9 +238,64 @@ We have the option to set an env variable in the config to use AWS spot instance
 > 
 > Spot instances are not to be used in production.
 
-##### Important Workflow.Next<!-- omit from toc -->
+##### firewatch-rosa-aws-sts-hypershift<!-- omit from toc -->
 
-As this level of testing expands to different flavors of OCP installations we will cover future important workflows here for reference.
+step-registry doc: [firewatch-rosa-aws-sts-hypershift](https://steps.ci.openshift.org/workflow/firewatch-rosa-aws-sts-hypershift)
+
+This workflow is going to serve as the base installation mechanism for our ROSA Hypershift OpenShift installation. This workflow can be used in a very similar way to how we use ipi-aws.
+
+Requirements:
+
+ - An AWS account that is setup to deploy ROSA on [staging env](https://qaprodauth.console.redhat.com/).
+
+AWS setup done by workflow:
+
+ - For a ROSA deployment we need things like a VPC, subnets, and roles created. This workflow does everything we need for the deployment for us so there is no need to create and manage anything prior to running this workflow. ([Pre steps](https://steps.ci.openshift.org/workflow/firewatch-rosa-aws-sts-hypershift#pre)).
+
+AWS cleanup done by workflow:
+
+ - This workflow also has a post section that will cleanup all AWS resources that were created as part of the workflows pre section ([Post steps](https://steps.ci.openshift.org/workflow/firewatch-rosa-aws-sts-hypershift#post)).
+
+Usage:
+
+ - There are many ways to use this workflow, so just to show a simple example we will show requirements to test ROSA Hypershift staging. (Please see the [Environment](https://steps.ci.openshift.org/workflow/rosa-aws-sts-hypershift#environment) section of the workflow to learn about these vars and more).
+
+Files created in the $SHARED_DIR
+
+ - The api.login file holds the username and password to login to the ROSA hypershift cluster.
+
+How to Access the ROSA Hypershift cluster:
```

Comment:
Can we move this down to this section or move it and link to this section here?

https://github.com/CSPI-QE/ocp-ci-docs/blob/main/docs/OCP_CI_Tutorials/Scenario_Development/Scenario_Development_Guide.md#accessing-the-test-cluster